### PR TITLE
Add alexarchambault to team members

### PR DESCRIPTION
**Pull request by alexarchambault**

### CODE REVIEW

File: _data/team-members.yml [hunk 1]
```diff
@@ -79,6 +79,21 @@ members:
        - title: dotty
          percentage: 10
     active: true
+  - member: alexandre
+    name: Alexandre Archambault
+    title: Software Development
+    email: alexandre.archambault@epfl.ch
+    github: alexarchambault
+    twitter: alxarchambault
+    photo:
```
Comment (sjrd) [inline _data/team-members.yml]: No photo? :cry:

**Comment by alexarchambault** [comment]
I left out planned features for coursier, I guess I'll add them once they get more precise.

I also left out the other stuff I'm working on (jupyter-related mostly). I'd like to see if I can get some support for [my own kernel](https://github.com/almond-sh/almond) from the other scala kernel authors, before mentioning it here.

File: _data/team-members.yml (hunk 1 above)
Comment (alexarchambault) [reply _data/team-members.yml]: Got to find a photo for ✨professional✨ contexts...

Comment (sjrd) [reply _data/team-members.yml]: OK, let's merge it as is for now, but I'll keep poking you until you've added a picture :)

**Comment by heathermiller** [comment]
It's OK to not have a great picture. Just something to help people identify you when you're at a conference. Doesn't have to be ✨amazing ✨hehe

**Comment by alexarchambault** [comment]
Promise I'll add a picture soon...!

**Pull request merged**